index.html

```diff
@@ -35,15 +47,40 @@ <h2>I'm a Software Developer</h2>
   </section>
 
   <section class="works">
```

Comment:
[ OPTIONAL ]Please align `Multi post stories` with the project after it horizontally. Please see below screenshot for reference.
![works](https://user-images.githubusercontent.com/35504149/130319849-b4146274-766e-44f8-ae9e-962b8793c18c.png)